rcon/index.js

```diff
@@ -362,4 +362,16 @@ export default class Rcon extends EventEmitter {
   decodedPacketToString(decodedPacket) {
     return util.inspect(decodedPacket, { breakLength: Infinity });
   }
+
+  async warn(steamID, message) {
+    await this.execute(`AdminWarn "${steamID}" ${message}`);
+  }
+
+  async kick(steamID, reason) {
+    await this.execute(`AdminKick "${steamID}" ${reason}`);
+  }
+
+  async switchTeam(steamID) {
```

Comment:
I'd keep these method names similar to the commands, i.e. `forceTeamChange`, on the basis this is what most Squad admins will know the commands as.